src/sync/installation.ts

```diff
@@ -207,7 +209,8 @@ const doProcessInstallation = async (data: BackfillMessagePayload, sentry: Hub,
 	const gitHubProduct = getCloudOrServerFromGitHubAppId(subscription.gitHubAppId);
 
 	if (!nextTask) {
-		await subscription.update({ syncStatus: "COMPLETE" });
+		const newBackFillSinceDate = calcNewBackfillSinceDate(subscription.backfillSince, data.commitsFromDate, rootLogger);
+		await subscription.update({ syncStatus: "COMPLETE", backfillSince: newBackFillSinceDate });
```

Comment:
We have TWO places for setting `COMPLETE` and metrics seems different! Which one is the one being used?